src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,48 @@
 
 class Animal {
   // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    if (Animal.alive == null) {
+      Animal.alive = [];
+    };
+
+    if (Animal.alive) {
+      Animal.alive.push(this);
+    }
+  }
 }
 
```

Comment:
```suggestion
  // write your code here
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
    Animal.alive.push(this);
  }

  static alive = [];
}

```

Reply:
when i add the array to static it gives an error "Parsing error: Unexpected token ="